stripper/ze_doom3_v2.cfg

```diff
@@ -156,4 +137,560 @@ modify:
+	}
+	insert:
+	{
+		"OnStartTouch" "!activatorAddContextzm_user:10-1"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "pinky_skin_stripweapon"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activatorAddContextzm_user:10-1"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "skin_spider_strip"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activatorAddContextzm_user:10-1"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "skin_revenant_strip"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activatorAddContextzm_user:10-1"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+		"hammerid" "135137"
+	}
+	insert:
+	{
+		
+		"OnMapSpawn" "playerClearContext0-1"
+	}
+}
+add:
+{
+	"origin" "-5792 12144 -4128"
+	"classname" "filter_activator_context"
+	"targetname" "filter_zm_user"
+	"ResponseContext" "zm_user"
+	"Negated" "Allow entities that match criteria"
+}
+add:
+{
+	"model" "*36"
+	"classname" "trigger_multiple"
+	"origin" "-6417.82 12201.3 -4066.5"
+	"filtername" "filter_zm_user"
+	"StartDisabled" "0"
+	"wait" "0"
+	"spawnflags" "1"
+	"OnStartTouch" "!activatorAddOutputorigin -6441 11506 -41200-1"
+}
+; Add tp avoid for stage 3 (start area) (3 spots)
+add:
+{
+	"classname" "trigger_teleport"
+	"targetname" "strp_stg3_tp_avoid"
+	"spawnflags" "1"
+	"origin" "-7110 14200 -3822.5"
+	"startdisabled" "1"
+	"target" "hellteleport"
+	"UseLandmarkAngles" "1"
+}
+add:
+{
+	"model" "*38"
+	"classname" "trigger_teleport"
+	"targetname" "strp_stg3_tp_avoid_2"
+	"spawnflags" "1"
+	"origin" "-8494.82 11267.6 -4021.5"
+	"startdisabled" "1"
+	"target" "hellteleport"
+	"UseLandmarkAngles" "1"
+}
+add:
+{
+	"model" "*203"
+	"targetname" "strp_stg3_tp_avoid_3"
+	"classname" "trigger_teleport"
+	"origin" "-3303 13052.1 -3903.08"
+	"spawnflags" "1"
+	"startdisabled" "1"
+	"target" "hellteleport"
+	"UseLandmarkAngles" "1"
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"hammerid" "2673"
+	}
+	insert:
+	{
+		"OnStartTouch" "strp_stg3_tp_avoidEnable101"
+		"OnStartTouch" "strp_stg3_tp_avoid_2Enable101"
+		"OnStartTouch" "strp_stg3_tp_avoid_3Enable101"
+	}
+}
+; Add tp to stage 2 that exists on css 
+add:
+{
+	"targetname" "strp_tp_stg2_gap"
+	"classname" "trigger_teleport"
+	"origin" "-5502 14562 -4044.5"
+	"spawnflags" "1"
+	"startdisabled" "1"
+	"target" "teler1"
+	"UseLandmarkAngles" "1"
+}
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"hammerid" "2786"
+	}
+	insert:
+	{
+		"onpressed" "strp_tp_stg2_gapEnable321"
+	}
+}
+; Nerf zm spider fireball (This nerf most likely needs tweaking)
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "hurtimp1"
+	}
+	replace:
+	{
+		"damage" "35"
+	}
+}
+; Nerf zm skeleton firebomb (This nerf most likely needs tweaking)
+modify:
+{
+	match:
+	{
+		"classname" "env_explosion"
+		"targetname" "gila3"
+	}
+	replace:
+	{
+		"iMagnitude" "30"
+		"iRadiusOverride" "350"
+	}
+}
```

Comment:
Same thing, start slower. Original radius with 45 magnitude is probably a good starting point.